install.sh

```diff
@@ -94,133 +104,125 @@ esac
+  extract_install "${name}" "${tarfile}"
+  update_device_json "${name}" "${version}"
 done
 
-# download, prepare and install chromebrew
-cd $CREW_LIB_PATH
-rm -rf crew lib packages
-curl -# -o crew $URL/crew
-chmod +x crew
-rm -f $CREW_PREFIX/bin/crew
-ln -s `pwd`/crew $CREW_PREFIX/bin
-#install crew library
-mkdir -p $CREW_LIB_PATH/lib
-cd $CREW_LIB_PATH/lib
-curl -# -o package.rb $URL/lib/package.rb
-curl -# -o package_helpers.rb $URL/lib/package_helpers.rb
-
-# Making GCC act like CC (For some npm packages out there, only required for gcc)
-#rm -f $CREW_PREFIX/bin/cc
-#ln -s $CREW_PREFIX/bin/gcc $CREW_PREFIX/bin/cc
-
-# package gcc7 has already made symbolic links for cc and gcc, no action required here
+# create symlink to 'crew' in ${CREW_PREFIX}/bin/
+rm -f "${CREW_PREFIX}/bin/crew"
+ln -s "../lib/crew/crew" "${CREW_PREFIX}/bin/"
 
 # prepare sparse checkout .rb packages directory and do it
-cd $CREW_LIB_PATH
+cd "${CREW_LIB_PATH}"
 rm -rf .git
 git init
-git remote add -f origin https://github.com/$OWNER/$REPO.git
+git remote add -f origin "https://github.com/${OWNER}/${REPO}.git"
 git config core.sparsecheckout true
 echo packages >> .git/info/sparse-checkout
 echo lib >> .git/info/sparse-checkout
 echo crew >> .git/info/sparse-checkout
-git fetch origin master
-git reset --hard origin/master
+git fetch origin "${BRANCH}"
+git reset --hard origin/"${BRANCH}"
 
 # install a base set of essential packages
 yes | crew install buildessential less most
 
+echo
+echo "If you have installed Chromebrew in a directory other than '/usr/local',"
+echo "please run these commands:"
+echo "echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc && . ~/.bashrc"
+echo "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc && . ~/.bashrc"
+echo "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc && . ~/.bashrc"
 echo
 echo "To set the default PAGER environment variable to be able to use less:"
-echo "echo \"export PAGER=$CREW_PREFIX/bin/less\" >> ~/.bashrc && . ~/.bashrc"
+echo "echo \"export PAGER='less'\" >> ~/.bashrc && . ~/.bashrc"
 echo
 echo "Alternatively, you could use most.  Why settle for less, right?"
-echo "echo \"export PAGER=$CREW_PREFIX/bin/most\" >> ~/.bashrc && . ~/.bashrc"
+echo "echo \"export PAGER='most'\" >> ~/.bashrc && . ~/.bashrc"
 echo
-echo "You may wish to set the default EDITOR.  The current EDITOR is '$EDITOR'."
-echo "For example, to set vim as the default editor, execute the following:"
-echo "echo \"export EDITOR=$CREW_PREFIX/bin/vim\" >> ~/.bashrc && . ~/.bashrc"
-echo "To install the vim editor, run 'crew install vim'"
+echo "You may wish to set the default EDITOR.  The current EDITOR is '${EDITOR}'."
+echo "For example, to set nano as the default editor, execute the following:"
+echo "echo \"export EDITOR='nano'\" >> ~/.bashrc && . ~/.bashrc"
+echo "To install the simple nano editor, run 'crew install nano'"
```

Comment:
The default `EDITOR` is already nano so that is why we added the vim example.  No need to change to nano since it would be redundant.

Reply:
Oh. I suppose that I could change that.

Reply:
I would recommend vim over nano but whatever.  An editor is a preference and should not be recommended.

Reply:
Yeah, although `nano` is easier to understand. A newbie user wouldn't be able to use `vim`.

Reply:
Oh, and by the way, `EDITOR` is set to `/bin/nano`, which just happens to be nonexistent. I'm reverting this.

Reply:
Well, it's really user choice. I'll just leave it.